unity/Assets/0Refactored/reusableGui/Menu/PrefabsColumns/scrMenuColumnLanguage.cs

```diff
@@ -0,0 +1,65 @@
+﻿using GEAR.Localization;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class scrMenuColumnLanguage : MonoBehaviour
+{
+    [SerializeField] private GameObject ButtonEnglish;
+
+    [SerializeField] private GameObject ButtonGerman;
+
+    [SerializeField] private Texture TextureIconBase;
+
+    [SerializeField] private Texture TextureIconSelected;
+
+
+    private void Start()
+    {
+        this.ButtonGerman.GetComponent<Button>().onClick.AddListener(() => this.OnClickGerman());
+        this.ButtonEnglish.GetComponent<Button>().onClick.AddListener(() => this.OnClickEnglish());
+        this.UpdateLocalizedText();
+        this.UpdateActiveButton();
+    }
+
+    private void OnClickGerman()
+    {
+        LanguageManager.Instance.CurrentLanguage = SystemLanguage.German;
+        this.UpdateLocalizedText();
+        this.UpdateActiveButton();
+    }
+
+    private void OnClickEnglish()
+    {
+        LanguageManager.Instance.CurrentLanguage = SystemLanguage.English;
+        this.UpdateLocalizedText();
+        this.UpdateActiveButton();
+    }
+
+    private void UpdateLocalizedText()
```

Comment:
We do not need this function.
There is an OnLanguageChange event where the LocalizedText is attached and updates itself when the language is changed.